spec/date_time_spec.rb

```diff
@@ -1,21 +1,17 @@
 require 'spec_helper'
 
 describe DateTime do
-
-  before do
-    @date_time = DateTime.parse("2010-01-01")
-  end
+  subject { DateTime.parse("2010-01-01") }
 
   it "should allow advancing by calendar days" do
-    @date_time.advance_considering_calendar(:calendar_days, 10).should eql DateTime.parse("2010-01-10 23:59:59")
+    expect(subject.advance_considering_calendar(:calendar_days, 10)).to eq DateTime.parse("2010-01-10 23:59:59")
   end
 
   it "should allow advancing by calendar months" do
-    @date_time.advance_considering_calendar(:calendar_months, 10).should eql DateTime.parse("2010-10-31 23:59:59")
+    expect(subject.advance_considering_calendar(:calendar_months, 10)).to eq DateTime.parse("2010-10-31 23:59:59")
   end
 
   it "should allow advancing by calendar years" do
-    @date_time.advance_considering_calendar(:calendar_years, 10).should eql DateTime.parse("2019-12-31 23:59:59")
+    expect(subject.advance_considering_calendar(:calendar_years, 10)).to eq DateTime.parse("2019-12-31 23:59:59")
```

Comment:
Line is too long. [113/80]<br>Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.